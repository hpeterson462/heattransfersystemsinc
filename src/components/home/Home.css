.logo-title {
  display: flex;
  flex-direction: column;
  justify-content: center;
  border-radius: 2px;
}

.home-logo {
  flex-direction: row;
  width: 125px;
  margin: auto;
  margin-top: 7%;
  border-radius: 2px;
}

.home-company-title {
  width: 70%;
  margin: auto;
  padding: 35px;
  border-radius: 2px;
  text-align: center;
  font-size: 4rem;
  font-weight: bolder;
}

.home-p {
  width: 50%;
  font-family: 'Noto Sans', sans-serif;
  border-radius: 2px;
  letter-spacing: 0.3rem;
  line-height: 2rem;
  font-weight: bolder;
  color: #003366;
}

@media only screen
and (min-device-width : 20em) 
and (max-device-width : 50em)
and (-webkit-min-device-pixel-ratio: 3)
and (orientation: portrait) {
  .home-logo {
    margin-top: 35%;
   }
}
