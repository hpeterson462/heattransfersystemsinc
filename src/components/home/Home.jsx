import React from 'react';
import Nav from '../nav/Nav';
import Translate from '../translate/Translate';
import Solutions from '../solutions/Solutions';
import Services from '../servicesPage/Services';
import Products from '../products/Products';
import AboutUs from '../aboutUs/AboutUs';
import Contact from '../contact/Contact';
// import Footer from '../footer/Footer';
import logo from '../../assets/logo.png';
import './Home.css';

export default function Home() {

  return (
    <>
      <Nav />

      <div className="logo-title" id="home">
        <img
          className="home-logo"
          src={logo}
          alt="Heat Transfer Systems logo"
        />
        <h1 className="home-company-title" data-testid="title">
          Heat Transfer Systems
        </h1>
        <h2 className="home-p">
          Burners, Combustion Systems, Components & Service
        </h2>
      </div>

      <Translate />

      <br />

      <Solutions />

      <br />

      <Services />

      <br />

      <Products />

      <br />

      <AboutUs />

      <br />

      <Contact />

      {/* <br />

      <Footer /> */}

    </>
  )
}
