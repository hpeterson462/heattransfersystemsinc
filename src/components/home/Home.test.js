import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Home from './Home';

describe('Home component', () => {
  afterEach(() => cleanup());

  it('renders Home component', () => {
    const { asFragment } = render(<MemoryRouter><Home /></MemoryRouter>);

    expect(asFragment()).toMatchSnapshot();
  });

  it('renders Home logo and title', () => {
    const { getByTestId } = render(<MemoryRouter><Home /></MemoryRouter>);

    expect(getByTestId('title').textContent).toBe('Heat Transfer Systems');
  })
});
