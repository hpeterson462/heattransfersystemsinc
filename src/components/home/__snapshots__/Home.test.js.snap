// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders Home component 1`] = `
<DocumentFragment>
  <nav
    class="navbar"
    style="top: 0px;"
  >
    <a
      href="/#home"
    >
      <img
        alt="small logo icon"
        class="logo-icon"
        src="logo.png"
      />
    </a>
    <a
      class="link"
      href="/#home"
    >
      Home
    </a>
    <a
      class="link"
      href="/#solutions"
    >
      Solutions
    </a>
    <a
      class="link"
      href="/#services"
    >
      Services
    </a>
    <a
      class="link"
      href="/#products"
    >
      Products
    </a>
    <a
      class="link"
      href="/#aboutus"
    >
      About Us
    </a>
    <a
      class="link"
      href="/#contact"
    >
      Contact
    </a>
  </nav>
  <div
    class="logo-title"
    id="home"
  >
    <img
      alt="Heat Transfer Systems logo"
      class="home-logo"
      src="logo.png"
    />
    <h1
      class="home-company-title"
      data-testid="title"
    >
      Heat Transfer Systems
    </h1>
    <h2
      class="home-p"
    >
      Burners, Combustion Systems, Components & Service
    </h2>
  </div>
</DocumentFragment>
`;
